execute/optimizers/type_optimizer.go

```diff
@@ -97,7 +97,7 @@ func (op ObservationOptimizer) TruncateObservation(observation exectypes.Observa
 					op.lggr.Errorw("missing message", "seqNum", seqNum, "chain", chain)
 					continue
 				}
-				op.lggr.Debugw("truncating message", "seqNum", seqNum, "chain", chain)
+				//op.lggr.Debugw("truncating message", "seqNum", seqNum, "chain", chain)
```

Comment:
So are we sure the logger is the cause? Removing these logs is problematic, especially if we suspect issues in the truncation algo